provider.go

```diff
@@ -309,6 +316,9 @@ func (g *InstanceGroup) Increase(
 			SSHKeys:      []string{string(publicKey)},
 			VolumeSizeGB: g.VolumeSizeGB,
 			UserData:     g.UserData,
+			Interfaces:   &[]cloudscale.InterfaceRequest{
```

Comment:
This should be `Interfaces:   &g.Interfaces,`, as it stands it won't compile (it's not ideal that we do not have a CI process for outside PRs yet, otherwise this would have been fairly obvious).